packages/protocol/src/LiquidationManager.sol

```diff
@@ -156,17 +180,20 @@ contract LiquidationManager is ILiquidationManager, SystemAccessControl {
    * @notice Callback function that completes execution logic of a liquidation
    * operation with a flashloan.
    *
-   * @param vault that holds user's position
    * @param users to be liquidated
+   * @param liqCloseFactors (optional) for each user, otherwise pass zero for each
```

Comment:
@DaigaroCota Can you better describe the `liqCloseFactor` in the comment and how it would affect calling this method with zeros or other values?

Reply:
I see actually it in the `IVault` but I guess giving an example could further help the caller in determining this value.

Reply:
Added an example as a dev doc on how to call `{LiquidationManager.liquidate(...)` with args using ethersJs.